dev/NavigationView/Strings/en-us/Resources.resw

```diff
@@ -157,4 +157,8 @@
     <value>More</value>
     <comment>Text for the nav view more button when panel is on top</comment>
   </data>
+  <data name="NavigationOverflowButtonToolTip" xml:space="preserve">
+    <value>See more</value>
+    <comment>ToolTip caption for the nav view more button when panel is on top</comment>
```

Comment:
Should "panel" be "pane" here? I matched the given descriptions for the NavigationView resources, such as here: https://github.com/microsoft/microsoft-ui-xaml/blob/d9a778927715af3dabeb21d6fc2152d56cb0f72f/dev/NavigationView/Strings/en-us/Resources.resw#L156

Reply:
My understanding is that this comment field is mostly used to give the localization team the context they need to make an accurate translations. There might be multiple ways of saying See more in a particular language and this helps them choose the correct one.

Reply:
I was more thinking this might have been a typo since "pane" is normally used for the NavigationView in this context (i.e. PaneDisplayMode).

Reply:
I think either word would achieve the same result.

Reply:
Let's leave it as it is then :)